hap-ibd/dockerfile

```diff
@@ -0,0 +1,50 @@
+################## BASE IMAGE ######################
+FROM adoptopenjdk:8-jre@sha256:bf736024c65942ab045c497e408706f9e599651bf8c01a75ecb3d6971d66700d
```

Comment:
Why not just using biocontainerd or ubuntu base image and install openjdk?

Reply:
I thought it would be more straightforward to use an image which already had java, but will try now with a different base image!